packages/lib/sdk/src/utils/svelte/buildUrl.js

```diff
@@ -0,0 +1,28 @@
+import { config } from '$evidence/config';
+/**
+ * Adjusts a path to include the configured base path
+ * Ignores undefined, and absolute URLs
+ * @param {string} path 
+ * @example buildUrl('http://localhost:3000/test') // 'http://localhost:3000/test'
+ * @example buildUrl('/test') // '/base/test'
+ * @example buildUrl(undefined) // undefined
+ * 
+ * @returns 
+ */
+export const buildUrl = (path) => {
+    if (path === undefined) return path;
+    if (path.startsWith('http')) return path;
+    
+    let basePath = config.deployment.basePath;
+    if (basePath) {
+        if (!basePath?.startsWith('/')) {
+            basePath = `/${basePath}`;
+        }
+        if (basePath.endsWith('/')) {
+            basePath = basePath.slice(0, -1);
+        }
+        return `${basePath}${path}`;
```

Comment:
This assumes that `path` always starts with `/`. Is this guaranteed to be true?

I think we should either
- Ensure append a `/` if `path` doesn't start with `/`, or
- Throw a descriptive error if `path` doesn't start with `/` rather than returning an incorrect URL (e.g. `/basetest` instead of `/base/test`)?